ci/.gitlab-ci.integration.yml

```diff
@@ -7,7 +7,7 @@ variables:
 
 build:docker-images:
   image: $BASE_DOCKER_IMAGE
-  stage: .pre
+  stage: .post
```

Comment:
this means that we are unable to ever use the image that we build in the same pipeline unless we run at at least once, is this something we want?

Reply:
We couldn't do that before either. This should be a seperate job for all the reasons i explained in the main channel

Reply:
The way to work around this would be to do any changes to the base docker image in isolation and then open a second PR with whatever you want to do with the improved image

Reply:
Yeah, this shouldn't be a separate thing